client/trino-client/src/main/java/okhttp3/internal/tls/DistinguishedNameParser.java

```diff
@@ -0,0 +1,426 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.internal.tls;
+
+import javax.security.auth.x500.X500Principal;
+
+// Formatted copy of https://raw.githubusercontent.com/square/okhttp/fc7ac8ae0b30d219cbd884b2af458f19c170c5fb/okhttp/src/main/java/okhttp3/internal/tls/DistinguishedNameParser.java
```

Comment:
Can you mention the reason to keep the copy here, and why you've chosen this particular commit? I tried figuring this out myself but it's not trivial. When I open https://github.com/square/okhttp/commit/fc7ac8ae0b30d219cbd884b2af458f19c170c5fb it is an unrelated change, and closest tags don't help:
![obraz](https://user-images.githubusercontent.com/795177/200109591-fd13afdb-e61a-4f21-9ce4-3efc784ef179.png)

Is this the last commit from the 3.x version we're using?

Reply:
Yes @nineinchnick. David pointed me to this one